apps/src/templates/GameButtons.story.jsx

```diff
@@ -1,111 +1,21 @@
 import React from 'react';
-import {UnconnectedGameButtons, RunButton, ResetButton} from './GameButtons';
+import {UnconnectedGameButtons} from './GameButtons';
 
-export default function (storybook) {
-  storybook.storiesOf('Buttons/GameButtons/RunButton', module).addStoryTable([
-    {
-      name: 'default',
-      description: 'The way the button is rendered with only default props',
-      story: () => <RunButton />,
-    },
-    {
-      name: 'minecraft version',
-      description: `
-          this shows up if you wrap the button in a ".minecraft" element
-          and include the appropriate minecraft stylesheet.
-        `,
-      storyCellClass: 'GameButtonsStory loadCraftStyleSheet',
-      story: () => (
-        <div className="minecraft">
-          <RunButton />
-        </div>
-      ),
-    },
-  ]);
+export default {
+  component: UnconnectedGameButtons,
+};
 
-  storybook.storiesOf('Buttons/GameButtons/ResetButton', module).addStoryTable([
-    {
-      name: 'default',
-      description:
-        'You have to explicitly set display: block to make this show up. It is hidden by default?!',
-      story: () => <ResetButton style={{display: 'block'}} />,
```

Comment:
You explained that the `ResetButton` is hidden by default and there's no mechanism via props to get it to show up thru GameButtons (in our actual code, we hide/show it via code in studioApp. Do you mind explaining where this happens?
I see in `TopInstructions.story.jsx`, [a `createCommonStore` function](https://github.com/code-dot-org/code-dot-org/blob/dc60f4c3a1df3ba491d6489c860754d48a1a5939/apps/src/templates/instructions/TopInstructions.story.jsx#L30) is used to initialize a Redux store and set `pageConstants` values. Is this possible for `pageConstants.hideResetButton` and `pageConstants.hideRunButton`?
I like your solution, but just curious if you think this approach would've also worked.

Reply:
haha yeah it is pretty confusing...so `ResetButton` is hidden by default per the `style` prop:

https://github.com/code-dot-org/code-dot-org/blob/5ccf9d443372bf05ee37b1846f1fdedd21334709/apps/src/templates/GameButtons.jsx#L40-L51

That `hideResetButton` prop gets passed as the `hidden` prop to `ResetButton`, which applies a css class that will hide the button, but not show it.

`GameButtons` doesn't pass a `style` prop to `ResetButton`, so you can't get it to show up via props from `GameButtons`. I could, of course, make this possible, but didn't want to make changes for the sake of Storybook (maybe I should?)

So, all of that to say, the Redux approach I don't think would work -- separately, I'd also generally advocate for us to use as little Redux as possible in Storybook.

Reply:
Ah and sorry, I think in our code this is how we hide/show the reset button:

https://github.com/code-dot-org/code-dot-org/blob/dc60f4c3a1df3ba491d6489c860754d48a1a5939/apps/src/StudioApp.js#L1112-L1116

Reply:
Oh interesting - thanks!
Do we need this line?
https://github.com/code-dot-org/code-dot-org/blob/5ccf9d443372bf05ee37b1846f1fdedd21334709/apps/src/templates/GameButtons.jsx#L48
Seems unnecessary since style is set below by `commonStyles.hidden` and updated accordingly in `StudioApp.js`.
Maybe a comment here would be helpful that styling is updated in `StudioApp`?

Reply:
I'm not really sure, I'll add a comment though. Here's a PR where Maddie got into this a bit if you want to know more:

https://github.com/code-dot-org/code-dot-org/pull/41118